core/parsigex/parsigex_test.go

```diff
@@ -39,11 +39,7 @@ func TestParSigEx(t *testing.T) {
 
 	pubkey := testutil.RandomCorePubKey(t)
 	data := core.ParSignedDataSet{
-		pubkey: core.ParSignedData{
-			Data:      []byte("partially signed data"),
-			Signature: nil,
-			ShareIdx:  0,
-		},
+		pubkey: core.NewParSig(testutil.RandomEth2Signature(), 0),
```

Comment:
`ParSig` is the simplest implementation, it only contains a signature and no data.

Reply:
Got it!